tests/cases/setup_hammerdb/vars.json

```diff
@@ -1,5 +1,5 @@
 {
   "use_hostname": false,
-  "hammerdb": "/root/HammerDB-4.6",
+  "hammerdb": "/home/hammerdb/HammerDB-4.6",
```

Comment:
Is this variable used anywhere?

Reply:
I believe it's used in this function: https://github.com/EnterpriseDB/edb-ansible/blob/83db607e08fdf327639822e6e5b8a3511ba7d94b/tests/tests/test_setup_hammerdb.py#L13

Reply:
I see, I don't like that its not consistent with vars used within the role but keep it as is, we should probably hard code the file location in the test script but that can be done another time.